### DISCUSSION TITLE
How to use UUI lib with NextJS 13 Router?

### DISCUSSION THREAD

**Discussion by alexbaumgertner** [epam/UUI#1540] [Q&A]
Hi!

There is a massive update of NextJS framework — application folder,

> "a new App Router built on [React Server Components](https://nextjs.org/docs/getting-started/react-essentials#server-components), which supports shared layouts, nested routing, loading states, error handling, and more."

https://nextjs.org/docs/app/building-your-application/routing

Do you have an example how to install UUI on NextJS 13?
It seems like current example https://github.com/epam/UUI/tree/main/next-app is based on old NextJS version.

**Comment by AlekseyManetov** [comment]
HI!
We didn't update our demo app on next.js 13 yet, but you can try to init project via our next.js template, which based on "13.1.5" version.
You can read how to init project with template here - https://uui.epam.com/documents?id=installation&mode=doc&skin=UUI4_promo
('Configuring from scratch for Next.js' section).
Template sources - https://github.com/epam/UUI/tree/main/templates/uui-nextjs-template
